cni.go

```diff
@@ -310,3 +312,27 @@ func (c *libcni) GetConfig() *ConfigResult {
 func (c *libcni) reset() {
 	c.networks = nil
 }
+
+// StatusDetail returns a slice of network statuses
+func (c *libcni) StatusDetail(ctx context.Context) ([]*NetworkStatus, error) {
+	err := c.Status()
+
+	if err != nil {
+		return nil, err
+	}
+
+	var networks []*NetworkStatus
+
+	for _, network := range c.Networks() {
+		if network.config.Name == "cni-loopback" {
```

Comment:
Why this specific magic name?  Can we move it out to a constant with a comment explaining?

Reply:
We are deprecating the cni-loopback and the return will always be the same regardless.